static/updating.md

```diff
@@ -22,6 +22,9 @@ Once the install has completed, you can then connect the EP1 to your WiFi easily
 {: .warning-title }
 If you do not see a "Connect" button below, use a supported web browser like Google Chrome.
 
+{: .warning-title }
+After clicking the "Connect" button, if the browser does not propose you "USB Serial ..." or you are having this error `Failed to execute 'open' on 'SerialPort': Failed to open serial port.`: please install the CH340 driver. The installer will propose you driver install links after trying to reconnect and not selecting a serial port.
```

Comment:
Would change the wording slightly to be:

After clicking the "Connect" button, if you do not see a "USB Serial" port listed, or you get the error "Failed to open serial port.", you may need to install the CH340 driver. The installer should give you links to the latest driver.

Reply:
```suggestion
After clicking the "Connect" button, if you do not see a "USB Serial" port listed, or you get the error "Failed to open serial port.", you may need to install the CH340 driver. The installer should give you links to the latest driver.
```

I'm good with that, I meen, I'm the French in the story here 😅